client/src/pages/AdminResourceManager.jsx

```diff
@@ -49,6 +50,21 @@ const AdminResourceManager = (props: Props): React$Element<'div'> => {
           </TabPane>
         </Tabs>
       </div>
+      <div>
+        <ModalView isAddCategory={true} isDeleteCategory={false}>
```

Comment:
These linting comments are essentially asking you to simplify the code by doing `<ModalView isAddCategory isDeleteCategory={false}>` - if you're passing a true value in, you don't need to specify it's true which is cool :)
(this comment is kind of irrelevant because of my above comment, but still useful to know)